### ISSUE TITLE
Schema Registry

### ISSUE THREAD

**Issue by kpathakota** [dendronhq/dendron#730]
## Context
Should be possible for users to easily create, discover and use existing schemas

## Proposal
Create a simple method (similar to pre-configured vaults) for accessing useful schemas.

**Comment by flammehawk** [comment]
I think the RFC for Schema Improvements #712 could be a valuable addition to this.

**Comment by kpathakota** [comment]
@flammehawk absolutely, I see them as related features (they both help each other).

**Comment by flammehawk** [comment]
Specially https://github.com/dendronhq/dendron-site/pull/89/files#diff-3956c364d18dd866588cd9d87abcaa45c5ea38d7d3af2b0d6f42a6938b83053d
that part of the RFC
